init.lua

```diff
@@ -32,6 +32,9 @@ local groups
 local mcl_hardness
 local mcl_blast_res
 local sounds
+local steel_ingot
+local wood
+local tree
```

Comment:
```suggestion
local steel_ingot, wood, tree
```

Reply:
Do you want the others in a row as well?

```local groups, mcl_hardness, mcl_blast_res, sounds```  
```local steel_ingot, wood, tree```

Reply:
Would be good IMO, but thats just style.

Reply:
Okay, done.